packages/in_app_purchase/example/lib/main.dart

```diff
@@ -0,0 +1,372 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_print
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:in_app_purchase/in_app_purchase.dart';
+import 'package:in_app_purchase_tizen/billing_manager_wrappers.dart';
+import 'package:in_app_purchase_tizen/in_app_purchase_tizen.dart';
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  InAppPurchaseTizenPlatform.registerPlatform();
+  runApp(_MyApp());
+}
+
+// To try without auto-consume, change `true` to `false` here.
+const bool _kAutoConsume = true;
+
+/// The [_kConsumableType]'s value:
+/// "1": CONSUMABLE
+/// "2": NON-CONSUMABLE
+/// "3": LIMITED-PERIOD
+/// "4": SUBSCRIPTION
+const int _kConsumableType = 1;
+const String _kAppId = '3201504002021';
```

Comment:
What does the app ID mean in this context? Isn't an "app ID" a string that looks like `org.tizen.my_app`?

Reply:
I explain `_kAppId` in [readme](https://github.com/hyue7/plugins/tree/in_app_purchase/packages/in_app_purchase#preparation) :
_kAppId: The APP ID in [APP Details Setting](https://dpi.samsungcheckout.com/settings/appdetails) when user  register and login to [Samsung Checkout DPI portal](https://dpi.samsungcheckout.com/)